.github/workflows/test.yml

```diff
@@ -21,7 +35,8 @@ jobs:
     - name: 'Prepare'
       shell: sh
       run: |
-        mkdir deps
+        mkdir -p deps/jtreg
+        mkdir -p /tmp/deps
```

Comment:
Looks like you moved `deps` to `/tmp`. Why is that?

Also, should the `deps/jtreg` also be moved to `tmp` in that case?

Reply:
there was a problem with relative paths on different platforms, so, replacing relative `.deps` to `/tmp/deps` works and feels better, the only leftover is `jtreg` because `action/checkout` can't checkout a project into a temp folder.